src/docs/automation-and-resiliency/request-quota-adjustment-for-openshift-project-set.md

```diff
@@ -219,9 +226,7 @@ If the request includes statistics on the resource usage of the front-end pods,
 
 ### Setting up resource monitoring with Sysdig Monitor
 
-Utilize Sysdig to monitor your application, accessing dashboards that display memory, CPU, and storage usage. 
-
-As explained in the [steps](#steps-to-request-quota-increase) to request a quota increase, it's crucial to monitor and collect metrics illustrating your application's resource utilization. 
+As explained in the [steps](#steps-to-request-quota-increase) to request a quota increase, it's crucial to monitor and collect metrics illustrating your application's resource utilization. Much of the information needed to justify a quota increase can be found in the OpenShift web console, but requires navigating through many different pages to collect a full view of the resource usage of your entire namespace. Sysdig makes the collection of this information much easier, for both the purposes of requesting a quota increase and for day-to-day namespace maintenance and monitoring.
 
 If you haven't already, check out the comprehensive guide on onboarding to application monitoring with Sysdig. Visit [Onboarding to application monitoring with Sysdig](../app-monitoring/sysdig-monitor-onboarding.md) for a step-by-step instructions for onboarding onto Sysdig and utilizing default dashboards to gain insights into your application's performance. For The documentation has all you need to onboard onto Sysdig and use the default dashboards.
 
```

Comment:
We also created a template called `Template - Resources Quota Approve Dashboard` that already exist in your sysdig dashboards list.